fast/stages/02-networking-nva/vpc-landing.tf

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# tfdoc:file:description Landing VPC and related resources.
+
+module "landing-project" {
+  source          = "../../../modules/project"
+  billing_account = var.billing_account_id
+  name            = "prod-net-landing-3" #TODO - reset counter
```

Comment:
Reminder - reset counter :)

Reply:
Yep, I'll look for all TODO as soon as we're ready to merge. Will be the last commit. I just want to make sure I can test everything until the merge :)

Reply:
Done. New commit coming.